nipype/interfaces/ants/registration.py

```diff
@@ -1596,3 +1596,59 @@ def _list_outputs(self):
             outputs['forward_warp_field'] = out_base + '1Warp.nii.gz'
             outputs['inverse_warp_field'] = out_base + '1InverseWarp.nii.gz'
         return outputs
+
+class CompositeTransformUtilInputSpec(ANTSCommandInputSpec):
+    process = traits.Enum('assemble', 'disassemble', argstr='--%s',
+        position=1, usedefault=True,
+        desc='What to do with the transform inputs (assemble or disassemble)',
+        )
+    in_file = InputMultiPath(File(exists=True), mandatory=True, argstr='%s...',
+        position=2, desc='Input transform file(s)')
+    output_prefix = Str("transform", usedefault=True, argstr='%s',
+        position=3, desc="A prefix that is prepended to all output files")
+
+class CompositeTransformUtilOutputSpec(TraitedSpec):
+    affine_transform = File(exists=True, desc="Affine transform component",
+            mandatory=True, position=2)
+    displacement_field = File(desc="Displacement field component")
+
+class CompositeTransformUtil(ANTSCommand):
+    """
+    ANTs utility which can combine or break apart transform files into their individual
+    constituent components.
+
+    Examples
+    --------
+
+    >>> from nipype.interfaces.ants import CompositeTransformUtil
```

Comment:
This is failing. For it to work, you'll need to import this interface into [`nipype/interfaces/ants/__init__.py`](https://github.com/nipy/nipype/blob/dcc013b0187bf5312db9174f8cb8932991c601cb/nipype/interfaces/ants/__init__.py#L7-L8):

https://github.com/nipy/nipype/blob/dcc013b0187bf5312db9174f8cb8932991c601cb/nipype/interfaces/ants/__init__.py#L7-L8